samples/listdirinfo.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
The double shebang lines here are unorthodox.  I suggest you remove the `/bin/bash` one, as that's not present by default on z/OS.